apps/www/_blog/2023-04-14-pg-tle.mdx

```diff
@@ -0,0 +1,66 @@
+
+Of course, TLEs _must_ be written in trusted languages as described above, hence the name, so no extensions that are written in C can be TLEs. There are still many extensions that are already written in trusted languages, and soon many more to come with some exciting progress in more performant trusted languages like [pl/rust](https://github.com/tcdi/plrust) which recently reached an official 1.0.0 version release. By removing the constraint that extensions must be installed on the server filesystem, TLEs open up a whole new world of possibilities for packaging and distributing new and modular functionality for Postgres.
+
+pg_tle is available on new Supabase projects and on Amazon RDS for Postgres 14.5 and higher. Supabase also includes an extension called [pgsql-http](https://github.com/pramsey/pgsql-http) for making HTTP calls from SQL to REST APIs. So, naturally putting two and two together, we thought it would be cool to see if it’s possible to install TLEs from a REST API using only SQL!
+
+You can, and we’ll show you how downloading TLEs directly into your database turns out not only to be possible but also very powerful! In this blog post we will present a prototype for this idea, and our future plans to extend TLEs into a full _community ecosystem_ of installable packages, just like many other languages enjoy. pip for Python, npm for Javascript, cargo for Rust, cpan for Perl, and now dbdev for Postgres!
+
+## The Extension Shortfall
+
+Because untrusted extensions are, by nature, able to break the security rules of Postgres, cloud providers are quite conservative about what extensions to provide to their customers. New extensions are rarely adopted quickly. This slows down the speed that new software can be created and shared. Open source requires opportunity and momentum, and by closing off the opportunity for Postgres users to quickly create, install, evaluate and iterate extensions, progress is slowed. TLEs offer an opportunity for open source developers to create and share code without having to wait for cloud providers or other gatekeepers to install it.
+
+After AWS released `pg_tle`, we realized immediately how powerful the tool is, and how it will change the way people release and install trusted extensions forever. After reaching out, we started collaborating with the TLE team at AWS and exchanging ideas and resources to further promote TLEs in both our database platform and theirs. By both Supabase and AWS providing the TLE extension in their respective Postgres platforms, we are unifying a standard package platform across two of the largest Postgres cloud providers in the world. We hope that other Postgres cloud providers will get on the bandwagon and provide TLE support as well, so that extension developers can leverage a large community base that greatly multiplies the reach of their efforts.
+
+## Installing dbdev with Migrations
```

Comment:
perhaps we should just link to the dbdev blog post? Make this one shorter

Reply:
maybe one better.. just link to database.dev "get going now.." sort of thing